pkgs/servers/wsdd/default.nix

```diff
@@ -1,28 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper, nixosTests, python3, fetchpatch }:
+{ lib, stdenv, fetchFromGitHub, makeWrapper, nixosTests, python3 }:
 
 stdenv.mkDerivation rec {
   pname = "wsdd";
-  version = "0.6.2";
+  version = "0.6.4";
 
   src = fetchFromGitHub {
     owner = "christgau";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0444xh1r5wd0zfch1hg1f9s4cw68srrm87hqx16qvlgx6jmz5j0p";
+    sha256 = "0lfvpbk1lkri597ac4gz5x4csfyik8axz4b41i03xsqv9bci2vh6";
   };
 
   nativeBuildInputs = [ makeWrapper ];
 
   buildInputs = [ python3 ];
 
-  patches = [
-    (fetchpatch {
-      # https://github.com/christgau/wsdd/issues/72
-      name = "fix_send_messages_using_correct_socket.patch";
-      url = "https://github.com/christgau/wsdd/commit/1ed74fe73a9fe2e2720859e2822116d65e4ffa5b.patch";
-      sha256 = "1n9bqvh20nhnvnc5pxvzf9kk8nky6rmbmfryg65lfmr1hmg676zg";
-    })
-  ];
+  patchPhase = ''
+    # Increase timeout to socket urlopen
+    # See https://github.com/christgau/wsdd/issues/80#issuecomment-76848906
+    substituteInPlace src/wsdd.py \
+      --replace '2.0' '5.0'
```

Comment:
Would it be possible to make this replacement a little more selective? I can foresee it catching some false positives and causing trouble at some point in the future.

Reply:
Replaced to patch.